src/Command/ErrorFormatter/RawErrorFormatter.php

```diff
@@ -19,13 +19,20 @@ public function formatErrors(
 			$output->writeLineFormatted('');
 		}
 
+		$outputIdentifiers = $output->isVerbose();
 		foreach ($analysisResult->getFileSpecificErrors() as $fileSpecificError) {
+			$identifier = '';
+			if ($outputIdentifiers && $fileSpecificError->getIdentifier() !== null) {
+				$identifier = sprintf(' [identifier:%s]', $fileSpecificError->getIdentifier());
```

Comment:
I'd rather not include the `:` here. If we omit that then we're just changing the format of the error message which is fine by me. But I' worried this `raw` error formatter is used by some parsers and this would be a BC break for them.

BTW I think this `raw` error formatter is really inferior - you always get absolute paths, and you don't get tips about errors.

Reply:
You're right, I fixed it!